touchdown/ssh/client.py

```diff
@@ -109,3 +142,4 @@ def connect(self, **kwargs):
             raise errors.Error("Unable to connect to remove server after {} tries".format(self.connection_attempts))
 
         self.verify_transport()
+        self.get_input_encoding()
```

Comment:
Total nit (and i'm guilty of this too) - i'd rather get_ methods returned something. Maybe ``detect_input_encoding`` or ``set_input_encoding``?